api/src/pcapi/core/mails/transactional/bookings/new_booking_to_pro.py

```diff
@@ -21,11 +21,14 @@ def get_new_booking_to_pro_email_data(
         event_date = ""
         event_hour = ""
 
-    if subcategories.ExtraDataFieldEnum.ISBN.value in offer.subcategory.conditional_fields:
-        isbn = offer.extraData.get("isbn", "") if offer.extraData is not None else ""
+    if (
+        subcategories.ExtraDataFieldEnum.EAN.value in offer.subcategory.conditional_fields
+        and offer.subcategoryId in subcategories.BOOK_WITH_EAN
```

Comment:
(question) Est-ce voulu de changer cette condition ? (En réalité, ça ne change rien, car les sous-catégories qui ont `EAN` dans leur `conditional_fields` sont les 3 qui sont dans `subcategories.BOOK_WITH_EAN`.)

Reply:
non y a le champs ean dans les conditional fields des CD aussi

Reply:
Oups, j'ai dû mal chercher. Dans le coup, est-ce qu'on ne pourrait pas simplifier la condition en `if offer.subcategoryId in subcategories.BOOK_WITH_EAN` ? Ça semble correspondre à l'intention : distinguer les livres.